config/nim.cfg

```diff
@@ -354,7 +354,7 @@ tcc.options.always = "-w"
   icl.cpp.options.always %= "${icl.cpp.options.always} /Qipo"
   gcc.options.always %= "${gcc.options.always} -flto=auto"
   gcc.cpp.options.always %= "${gcc.cpp.options.always} -flto=auto"
-  gcc.options.linker %= "${gcc.options.linker} -flto=auto"
+  gcc.options.linker %= "${gcc.options.linker} -flto=auto -Wno-stringop-overflow"  # https://github.com/nim-lang/Nim/issues/21595
```

Comment:
And what if the gcc version does not understand `-Wno-stringop-overflow`? We support pretty old GCCs too...

Reply:
It was added in GCC 6, we are at GCC 13+.